Compilation error when elf_cleaner not exist

In clean_elf function of build.py, the command to compile the binary native/out/elf-cleaner missed necessary cxxflags:

```python
         if not os.path.exists(elf_cleaner):
             execv(['g++', 'tools/termux-elf-cleaner/termux-elf-cleaner.cpp',
                   '-o', elf_cleaner])
```

Lacking of a "-std=c++11" flag caused the following compilation errors:

```
 Building binaries: magisk magiskinit magiskboot busybox

tools/termux-elf-cleaner/termux-elf-cleaner.cpp: In function ‘bool process_elf(uint8_t*, size_t, const char*)’:
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:77:32: error: ‘nullptr’ was not declared in this scope
     char const* removed_name = nullptr;
                                ^
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:102:50: error: expected ‘;’ before ‘orig_d_val’
      decltype(dynamic_section_entry->d_un.d_val) orig_d_val =
                                                  ^
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:104:50: error: expected ‘;’ before ‘new_d_val’
      decltype(dynamic_section_entry->d_un.d_val) new_d_val =
                                                  ^
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:106:10: error: ‘new_d_val’ was not declared in this scope
      if (new_d_val != orig_d_val) {
          ^
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:106:23: error: ‘orig_d_val’ was not declared in this scope
      if (new_d_val != orig_d_val) {
                       ^
tools/termux-elf-cleaner/termux-elf-cleaner.cpp: In instantiation of ‘bool process_elf(uint8_t*, size_t, const char*) [with ElfHeaderType = Elf32_Ehdr; ElfSectionHeaderType = Elf32_Shdr; ElfDynamicSectionEntryType = Elf32_Dyn; uint8_t = unsigned char; size_t = long unsigned int]’:
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:177:84:   required from here
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:102:14: error: ‘decltype’ was not declared in this scope
      decltype(dynamic_section_entry->d_un.d_val) orig_d_val =
              ^
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:104:14: error: ‘decltype’ was not declared in this scope, and no declarations were found by argument-dependent lookup at the point of instantiation [-fpermissive]
      decltype(dynamic_section_entry->d_un.d_val) new_d_val =
              ^
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:102:14: note: ‘decltype’ declared here, later in the translation unit
      decltype(dynamic_section_entry->d_un.d_val) orig_d_val =
              ^
tools/termux-elf-cleaner/termux-elf-cleaner.cpp: In instantiation of ‘bool process_elf(uint8_t*, size_t, const char*) [with ElfHeaderType = Elf64_Ehdr; ElfSectionHeaderType = Elf64_Shdr; ElfDynamicSectionEntryType = Elf64_Dyn; uint8_t = unsigned char; size_t = long unsigned int]’:
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:179:84:   required from here
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:102:14: error: ‘decltype’ was not declared in this scope
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:104:14: error: ‘decltype’ was not declared in this scope, and no declarations were found by argument-dependent lookup at the point of instantiation [-fpermissive]
      decltype(dynamic_section_entry->d_un.d_val) new_d_val =
              ^
tools/termux-elf-cleaner/termux-elf-cleaner.cpp:102:14: note: ‘decltype’ declared here, later in the translation unit
      decltype(dynamic_section_entry->d_un.d_val) orig_d_val =
```

A possible fix is to add a "-std=c++11" flag to the compilation command:

```python
        if not os.path.exists(elf_cleaner):
            execv(['g++', '-std=c++11', 'tools/termux-elf-cleaner/termux-elf-cleaner.cpp',
                  '-o', elf_cleaner])
```

--- a/build.py
+++ b/build.py
@@ -172,7 +172,7 @@ def clean_elf():
     else:
         elf_cleaner = os.path.join('native', 'out', 'elf-cleaner')
         if not os.path.exists(elf_cleaner):
-            execv(['g++', 'tools/termux-elf-cleaner/termux-elf-cleaner.cpp',
+            execv(['g++', '-std=c++11', 'tools/termux-elf-cleaner/termux-elf-cleaner.cpp',
                   '-o', elf_cleaner])
     args = [elf_cleaner]
     args.extend(os.path.join('native', 'out', arch, 'magisk') for arch in archs + arch64)

